building apvlv on nixos-unstable fails

## Issue description
building apvlv on nixos-unstable  22b7449aacb9bd064537d230176154292a2f2fec fails with
```
In file included from /nix/store/gs5zdwrvvfk6gzl5bmdanaf27pl37zhd-gtk+3-3.24.2-dev/include/gtk-3.0/gtk/gtkapplication.h:27:0,
                 from /nix/store/gs5zdwrvvfk6gzl5bmdanaf27pl37zhd-gtk+3-3.24.2-dev/include/gtk-3.0/gtk/gtkwindow.h:33,
                 from /nix/store/gs5zdwrvvfk6gzl5bmdanaf27pl37zhd-gtk+3-3.24.2-dev/include/gtk-3.0/gtk/gtkdialog.h:32,
                 from /nix/store/gs5zdwrvvfk6gzl5bmdanaf27pl37zhd-gtk+3-3.24.2-dev/include/gtk-3.0/gtk/gtkaboutdialog.h:30,
                 from /nix/store/gs5zdwrvvfk6gzl5bmdanaf27pl37zhd-gtk+3-3.24.2-dev/include/gtk-3.0/gtk/gtk.h:31,
                 from /build/source/src/ApvlvUtil.h:32,
                 from /build/source/src/ApvlvDoc.cc:29:
/nix/store/gs5zdwrvvfk6gzl5bmdanaf27pl37zhd-gtk+3-3.24.2-dev/include/gtk-3.0/gtk/gtkwidget.h:1144:14: note: declared here
 void         gtk_widget_override_color            (GtkWidget     *widget,
              ^~~~~~~~~~~~~~~~~~~~~~~~~
[ 33%] Building CXX object src/CMakeFiles/apvlv.dir/ApvlvFile.cc.o
[ 40%] Building CXX object src/CMakeFiles/apvlv.dir/ApvlvPdf.cc.o
[ 46%] Building CXX object src/CMakeFiles/apvlv.dir/ApvlvInfo.cc.o
/build/source/src/ApvlvPdf.cc:32:10: fatal error: goo/gtypes.h: No such file or directory
 #include <goo/gtypes.h>
          ^~~~~~~~~~~~~~
compilation terminated.
make[2]: *** [src/CMakeFiles/apvlv.dir/build.make:128: src/CMakeFiles/apvlv.dir/ApvlvPdf.cc.o] Error 1
make[2]: *** Waiting for unfinished jobs....
make[1]: *** [CMakeFiles/Makefile2:91: src/CMakeFiles/apvlv.dir/all] Error 2
make: *** [Makefile:152: all] Error 2
note: keeping build directory '/tmp/nix-build-apvlv-0.1.5.drv-0'
builder for '/nix/store/dkrdlxfci7509mzxw4wyxx4f1npabbrb-apvlv-0.1.5.drv' failed with exit cod

```
### Steps to reproduce

nix-build -A apvlv ~/nixpkgs

CC @ardumont

--- a/pkgs/applications/misc/apvlv/default.nix
+++ b/pkgs/applications/misc/apvlv/default.nix
@@ -41,6 +41,7 @@ stdenv.mkDerivation rec {
       url = "https://github.com/naihe2010/apvlv/commit/a3a895772a27d76dab0c37643f0f4c73f9970e62.patch";
       sha256 = "1fpc7wr1ajilvwi5gjsy5g9jcx4bl03gp5dmajg90ljqbhwz2bfi";
     })
+    ./fix-build-with-poppler-0.73.0.patch
   ];
 
   installPhase = ''

--- a/pkgs/applications/misc/apvlv/fix-build-with-poppler-0.73.0.patch
+++ b/pkgs/applications/misc/apvlv/fix-build-with-poppler-0.73.0.patch
@@ -1,0 +1,13 @@
+diff --git a/src/ApvlvPdf.cc b/src/ApvlvPdf.cc
+index 765b112..83d133f 100644
+--- a/src/ApvlvPdf.cc
++++ b/src/ApvlvPdf.cc
+@@ -29,7 +29,7 @@
+ #include "ApvlvPdf.h"
+
+ #ifndef POPPLER_WITH_GDK
+-#include <goo/gtypes.h>
++#include <goo/gfile.h>
+
+ static void
+ copy_cairo_surface_to_pixbuf (cairo_surface_t *surface,

